lib/active_merchant/billing/gateways/epsilon/epsilon_base.rb

```diff
@@ -68,7 +68,12 @@ def commit(path, request_params, response_keys = DEFAULT_RESPONSE_KEYS)
       end
 
       def post_data(parameters = {})
-        parameters.map { |k, v| "#{k}=#{CGI.escape(v.to_s)}" }.join('&')
+        parameters.map { |k, v| "#{k}=#{CGI.escape(encode_value(v.to_s))}" }.join('&')
+      end
+
+      def encode_value(value)
+        return value unless self.encoding
```

Comment:
@kurotaky Thank you. I fixed it. https://github.com/pepabo/active_merchant-epsilon/pull/82/commits/33f782f546fe0def6d7f9f93c7744ab9c84b623e